OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json

```diff
@@ -0,0 +1,80 @@
+{
+    "tzTU3gw6xGsWXYUdMkjjb:3:CL:27618:ontario-test-credential": {
+        "capture_base": {
+          "attributes": {
+            "name": "Text",
+            "civic_address": "Text",
+            "city": "Text",
+            "province": "Text",
+            "country": "Text",
+            "postal_code": "Text",
+            "birth_dateint": "Text",
+            "issue_date": "Text",
+            "expiry_date": "Text",
+            "test_id": "Text"
+          }
+        },
+        "overlays": [
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
+            "type": "spec/overlays/meta/1.0",
+            "language": "en",
+            "name": "Student"
+          },
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
+            "type": "spec/overlays/meta/1.0",
+            "language": "fr",
+            "name": "Étudiant"
+          },
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
+            "type": "spec/overlays/label/1.0",
+            "language": "en",
+            "attr_labels": {
+              "name": "Full Name",
+              "civic_address": "Address",
+              "city": "City",
+              "province": "Province",
+              "country": "Country",
+              "postal_code": "Postal Code",
+              "birth_dateint": "Date of birth",
+              "issue_date": "Issue Date",
+              "expiry_date": "Expiry",
+              "test_id": "Test ID"
+            }
+          },
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
+            "type": "spec/overlays/label/1.0",
+            "language": "fr",
+            "attr_labels": {
+              "name": "Nom et prénom",
+              "civic_address": "Adresse",
+              "city": "Ville",
+              "province": "Province",
+              "country": "Pays",
+              "postal_code": "Code Postal",
+              "birth_dateint": "Date de naissance",
+              "issue_date": "Date d'émission",
+              "expiry_date": "Expiration",
+              "test_id": "Test ID"
+            }
+          },
+          {
+            "capture_base": "EfeQBz5tyC0gett-ETgkg4pkxBwwuQoeYsT-9Nt0cNsc",
```

Comment:
This is not the “branding” overlay that we have defined in Aries.  The I’m not sure if “card_overlay” is a OCA overlay or not, but it doesn’t match what is expected by Aries wallets.